### ISSUE TITLE
UDTable inside a Modal causes a horizontal scroll to appear no matter the table size

### ISSUE THREAD

**Issue by DJS91** [ironmansoftware/powershell-universal#3166]
### Version

4.2.13

### Severity

Low

### Steps to Reproduce

![image](https://github.com/ironmansoftware/issues/assets/102936437/64eb8cf0-25f1-497c-ac97-9cc3d5e040c9)

showing a table inside a modal causes a small horizontal scroll to appear, seems there is some padding or margin misalignment. Styling either element doesn't seem to affect it.

Code:
`Show-UDModal -PaperStyle @{ overflowY = "visible" } -Content {
          
      $Lockedusers = $users | ?{$_.Lockedout -eq $true -and $_.Enabled -eq $true}

      New-UDTable -Data $Lockedusers -Icon (New-UDIcon -Icon 'User' -Style @{marginRight="10px"}) -Dense -ShowSort -ShowPagination -Title "Locked Out" -PageSize 10 -PaginationLocation top -Columns @(
          New-UDTableColumn -Property Samaccountname -Title "Samaccountname" -Filter
          New-UDTableColumn -Property Name -Title "Name" -Filter
          New-UDTableColumn -Property LockedOut -Title "Locked Out"
      ) 
  
  }`

### Expected behavior

```console
the modal should expand to fit the table rendered within without showing a scroll
```

### Actual behavior

```console
modal always shows a scroll and doesn't fully expand to fit the table element.
```

### Additional Environment data

Latest version of PSU
Browser: Edge but happens on all

### Visuals

https://github.com/ironmansoftware/issues/assets/102936437/a4a98f3a-9026-46b8-bfab-d26c9e2406ad

**Comment by kodifan** [comment]
yes i noticed this as well. i handle it by putting table inside New-UDPaper. then this scrollbar is gone

**Comment by DJS91** [comment]
> yes i noticed this as well. i handle it by putting table inside New-UDPaper. then this scrollbar is gone

Thanks for the workaround! 

Still a bug though!

**Comment by RVL-Cubics** [comment]
What happens if you use the -RemoveCard parameter?
